ep/lh/ep-lh-0009.md

```diff
@@ -0,0 +1,140 @@
+| Authors      | binore                                  |
+| Status       | Draft                                   |
+
+## Abstract
+
+This document describes the `lhman` - tool that allows to request engine's catalog, install engines and prepare necessary environment.
+
+## Motivation
+
+linterhub is a catalog that contains the information necessary to install, run and process the result of the work of the `linter`. To start working, you need a utility that allows you to centrally install the linter from the existing catalog.
+
+## Proposal
+
+lhman is a utility that allows you to obtain the catalog of linters and install them. For the implementation of the functional lhman is divided into `strategies`.
+
+## Strategies
+
+lhman implements the following strategies:
+
+| Name         | Description |
+| -            | -           |
+| catalog      | Get engines catalog |
+| install      | Install engine (or multiple engines) |
+| help         | Show help   |
+| version      | Get lhman version |
+
+> By default, space is used as delimiter between argument and value. The equal sign is also admissible. If the argument is at the same time several values, then it is recommended to use the comma sign to separate them. In addition, multiple declaration of arguments is allowed as an alternative. Example: `lhman strategy --arg1=value1,value2 --arg2 value1 --arg2=value2`
+
+### catalog
+
+This strategy generates 
+
+| Argument    | Type     | Required | Description |
+| -           | :-:      | :-:      | -           |
+| filter      | string   | -        | Engine name (or engine id) to install [ep-lh-0002](#related) |
+| keys        | string   | -        | Engine version (latest version by default) |
+| bundle      | string   | -        | Generate a bundle with `metadata`, `arguments` and `dependencies` for all engines. Argument requires file name as string. |
+| engine      | string   | -        | Engine name to return `metadata`, `arguments` and `dependencies` |
+| project     | string   | -        | Project path to return list of recommended for installing engines |
+
+Here is command to get only `name` and `description` properties of each supported engine.
+
+`lhman catalog --filter=name,description`
+
+```json
+{
+    "linters": [
+        {
+            "name": "alex",
+            "description": "Catch insensitive, inconsiderate writing.",
+        },
+        {
+            "name": "bandit",
+            "description": "Security oriented static analyser for python code.",
+        },
+        ...
+    ]
+}
+```
+
+### install
+
+The install strategy allows to download engine and prepare necessary environment. 
+
+| Argument    | Type     | Required | Description |
+| -           | :-:      | :-:      | -           |
+| engine      | string   | +        | Engine name (or engine id) to install [ep-lh-0002](#related) |
+| version     | string   | -        | Engine version (latest version by default) |
```

Comment:
We can't promise that latest version will be always supported, `version from catalog` by default.